Cargo.toml

```
@@ -1,7 +1,7 @@
 [package]
 name = "async_http_range_reader"
 authors = ["Bas Zalmstra <zalmstra.bas@gmail.com>"]
-version = "0.8.0"
+version = "0.8.1"
```

this is done by our release script

```suggestion
version = "0.8.0"
```